examples/classification/classification/workflow.py

```diff
@@ -0,0 +1,201 @@
+# Copyright 2021-2023 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import dataclasses
+from typing import List
+from typing import Optional
+
+from pydantic.dataclasses import dataclass
+
+from kolena.workflow import define_workflow
+from kolena.workflow import EvaluatorConfiguration
+from kolena.workflow import GroundTruth as BaseGroundTruth
+from kolena.workflow import Image
+from kolena.workflow import Inference as BaseInference
+from kolena.workflow import Metadata
+from kolena.workflow import MetricsTestCase
+from kolena.workflow import MetricsTestSample
+from kolena.workflow import MetricsTestSuite
+from kolena.workflow.annotation import ClassificationLabel
+from kolena.workflow.annotation import ScoredClassificationLabel
+
+
+@dataclass(frozen=True)
+class TestSample(Image):
+    """Test sample type for Classification workflow."""
+
+    metadata: Metadata = dataclasses.field(default_factory=dict)
+    """The metadata associated with the test sample."""
+
+
+@dataclass(frozen=True)
+class GroundTruth(BaseGroundTruth):
+    """Ground truth type for Classification workflow."""
+
+    classification: Optional[ClassificationLabel]
+    """The classfication label associated with an image. For binary classification, the negative class is `None`."""
+
+
+@dataclass(frozen=True)
+class Inference(BaseInference):
```

Comment:
Do we want an `ignore` field, similar to what got added for 2D OD?

Reply:
Ah I did add it on the latest branch.  Will be added in a subsequent PR!